proto/src/determined/workspace/v1/workspace.proto

```diff
@@ -97,18 +95,42 @@ message PatchWorkspace {
   string default_aux_pool = 15 [deprecated = true];
   // Name of the default aux pool can be optional.
   optional string default_aux_resource_pool = 17;
-  // Optional cluster namespace map to create a workspace namespace binding for
-  // each cluster.
-  map<string, string> cluster_namespace_pairs = 3;
+  // The map of clusters to workspace-namespace metadata (including the
+  // workspace-namespace binding and the resource quota placed on the namespace
+  // bound to the workspace for a given cluster).
+  map<string, determined.workspace.v1.WorkspaceNamespaceMeta>
+      cluster_namespace_meta = 18;
 }
 
 // WorkspaceNamespace represents a workspace-namespace binding for a given
-// cluster.
-message WorkspaceNamespace {
+// workspace and cluster.
+message WorkspaceNamespaceBinding {
```

Comment:
can we add auto_namespace to this and return this in ListWorkspaceNamespaceBindingsEndPoint? Alternatively, return WorkspaceNamespaceMeta